### ISSUE TITLE
modal, custom animation

### ISSUE THREAD

**Issue by samanmohamadi** [uikit/uikit#3159]
it would be nice if there was a possibility to use another animation for modals.
for example, **Scale Up**

**Comment by garethwarry** [comment]
Agreed, or having a faster default animation using `uk-animation-fast`.
